profiles/Search.py

```diff
@@ -0,0 +1,29 @@
+from .scraper import LinkedInScraper
+import time
+import urllib.request
+import os
+import json
+
+class Search:
+    currentPage = 0
+    
+    # LinkedIn Search is restricted to certain amount of people,
+    # Can use google to search instead.
+    def get_linkedin_urls_google_search(self, company, title):
```

Comment:
Doesn't have num_profiles

Reply:
I think this comment was made on a previous version of my code for some reason, but In the current iteration the class keeps track of which page it's on and returns the next 10 suggestions each time you call the function. Is that not what you wanted?